aquality-selenium-template/src/main/java/aquality/selenium/template/configuration/Environment.java

```diff
@@ -6,22 +6,15 @@
 import java.nio.file.Path;
 import java.nio.file.Paths;
 
-public enum Environment {
-    PROD("prod"),
-    STAGE("stage");
+class Environment {
 
-    private ISettingsFile configFile;
-
-    Environment(String name) {
-        this.configFile = getConfigFile(name);
+    private Environment() {
     }
 
-    private ISettingsFile getConfigFile(final String envName) {
+    static ISettingsFile getCurrentEnvironment() {
+        String envVarValue = System.getProperty("environment");
+        String envName = envVarValue == null ? "stage" : envVarValue;
```

Comment:
the same question as for .net: do not we want to move env value into settings.json?